src/fontra/views/editor/edit-tools-shape.js

```diff
@@ -0,0 +1,211 @@
+import * as rectangle from "../core/rectangle.js";
+import { range } from "../core/utils.js";
+import { VarPackedPath, packContour } from "../core/var-path.js";
+import { BaseTool, shouldInitiateDrag } from "./edit-tools-base.js";
+import { registerVisualizationLayerDefinition } from "./visualization-layer-definitions.js";
+
+export class ShapeToolRect extends BaseTool {
+  iconPath = "/tabler-icons/square-plus-2.svg";
+  identifier = "shape-tool-rectangle";
+
+  handleHover(event) {}
```

Comment:
This is redundant, and overwritten below.

Reply:
Thanks. done.